src/index.js

```diff
@@ -2,13 +2,13 @@ import React from 'react'
 import ReactDOM from 'react-dom'
 
 import App from './containers/App'
-import * as serviceWorker from './serviceWorker'
+import ErrorBoundary from './component/errorboundary/ErrorBoundary'
```

Comment:
Please, avoid default exports.

Reply:
In a nutshell, is to move the ErrorBoundary to the App.js, right now you call it in the index file.

What you do with that is create hierarchy over the application (App.js). I'm not saying it doesn't work. But nothing should be above App.js except <div.root>.

The exports is to avoid errors such as importing a component with a type or increasing the size of the Bundle Size because the default exports do not support treeshaking (remove dead code).

Translated with www.DeepL.com/Translator